main.nf

```diff
@@ -205,7 +205,7 @@ bundle_for_average
     .groupTuple(by: 0)
     .set{all_bundle_for_average}
 process Average_Bundles {
-    publishDir = "./results_rbx/Average_Bundles/"
```

Comment:
@frheault any reason to publish Average_Bundles in results_rbx instead of output_dir when necessary ?

Reply:
No reason, my mistake.